ui/src/pages/View.js

```diff
@@ -17,9 +17,106 @@
 import React, {Component} from 'react';
 import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
 
+import Footer from './../components/Footer.js';
 import LoadingIndicator from './../components/LoadingIndicator.js';
+import RepoHeader from './../components/RepoHeader.js';
+import SearchBar from './../components/SearchBar.js';
 import Utils from './../Utils.js';
 
+// TODO(gimite): Consider sharing some messages with Create.js because they
+// have many overlaps.
+const MESSAGES = defineMessages({
+  age: {
+    id: 'View.age',
+    defaultMessage: 'Age',
+    description: 'A label for a person\'s age.',
+  },
+  authorOfThisRecord: {
+    id: 'View.authorOfThisRecord',
+    defaultMessage: 'Author of this record',
+    description: ('The title of a section in a person record which contains '
+        + 'information of the author of the record.'),
+  },
+  city: {
+    id: 'View.city',
+    defaultMessage: 'City',
+    description: 'A label for a person\'s city.',
+  },
+  country: {
+    id: 'View.country',
+    defaultMessage: 'Country',
+    description: 'A label for a person\'s country.',
+  },
+  description: {
+    id: 'View.description',
+    defaultMessage: 'Description',
+    description: ('The title of a section in a person record which contains '
+        + 'free text description of the person.'),
+  },
+  homeAddress: {
+    id: 'View.homeAddress',
+    defaultMessage: 'Home address',
+    description: 'A label for a person\'s home address.',
+  },
+  identifyingInformation: {
+    id: 'View.identifyingInformation',
+    defaultMessage: 'Identifying information',
+    description: 'The title of a section which contains a person\'s '
+        + 'identifying information (name, age, etc.).'
+  },
+  notesForThisPerson: {
+    id: 'View.notesForThisPerson',
+    defaultMessage: 'Notes for this person',
+    description: ('The title of a section which contains notes for the person '
+        + 'record.'),
```

Comment:
Maybe "which contains notes with updates or additional information about the status of the person described on the page"?